css/wallscreens/sul-colors.css

```diff
@@ -12,6 +12,11 @@ body {
   filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
 }
 
+.header-logo {
+  min-height: 35px;
+  height: 3vw;
+}
```

Comment:
i don't think this thing actually needs to bother with grid/flex since its sibling (the title) should ignore it for purposes of being centered

```suggestion
  position: absolute;
  left: 50px;
  max-height: 40px;
}
```